src/DssSpell.sol

```diff
@@ -37,7 +81,42 @@ contract DssSpellAction is DssAction {
     // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/d8c711404e9e17db24b611aa03d45507c1993148/governance/votes/Executive%20vote%20-%20November%2016%2C%202022.md -q -O - 2>/dev/null)"
 
     string public constant override description =
-        "2022-11-16 MakerDAO Executive Spell | Hash: 0xf3023b0a9b3c423edf28bf4cdbd61d287ab29265d462a5fae0378227c153ea7b";
+        "2022-11-30 MakerDAO Executive Spell | Hash: TODO";
+
+    uint256 constant internal RAY = 10 ** 27;
+    uint256 constant internal RAD = 10 ** 45;
+    uint256 constant internal MILLION = 10 ** 6;
+
+    bytes32 constant internal ILK = "DIRECT-COMPV2-DAI";
+
+    address constant internal D3M_HUB = 0x12F36cdEA3A28C35aC8C6Cc71D9265c17C74A27F;
+    address constant internal D3M_MOM = 0x1AB3145E281c01a1597c8c62F9f060E8e3E02fAB;
+    address immutable internal D3M_MOM_LEGACY = DssExecLib.getChangelogAddress("DIRECT_MOM");
+    address constant internal D3M_COMPOUND_POOL = 0x621fE4Fde2617ea8FFadE08D0FF5A862aD287EC2;
+    address constant internal D3M_COMPOUND_PLAN = 0xD0eA20f9f9e64A3582d569c8745DaCD746274AEe;
+    address constant internal D3M_ORACLE = 0x0e2bf18273c953B54FE0a9dEC5429E67851D9468;
+    address constant internal D3M_CDAI = 0x5d3a536E4D6DbD6114cc1Ead35777bAB948E3643;
+    address constant internal D3M_COMPTROLLER = 0x3d9819210A31b4961b30EF54bE2aeD79B9c9Cd3B;
+    address constant internal D3M_COMP = 0xc00e94Cb662C3520282E6f5717214004A7f26888;
+    address constant internal D3M_TACK = 0xFB564da37B41b2F6B6EDcc3e56FbF523bD9F2012;
+    address constant internal D3M_DELEGATE = 0x3363BAe2Fc44dA742Df13CD3ee94b6bB868ea376;
+
+    // target 2% borrow apy, see top of D3MCompoundPlan for the formula explanation
+    // ((2.00 / 100) + 1) ^ (1 / 365) - 1) / 7200) * 10^18
+    uint256 constant internal D3M_COMP_BORROW_RATE = 7535450719;
+
+    address constant internal MCD_CLIP_CALC_GUSD_A = 0xC287E4e9017259f3b21C86A0Ef7840243eC3f4d6;
+    address constant internal MCD_CLIP_CALC_USDC_A = 0x00A0F90666c6Cd3E615cF8459A47e89A08817602;
+    address constant internal MCD_CLIP_CALC_PAXUSD_A = 0xA2a4aeFEd398661B0a873d3782DA121c194a0201;
+
+    address constant internal RETH_ORACLE = 0xF86360f0127f8A441Cfca332c75992D1C692b3D1;
+    address constant internal RETH_LIGHTFEED = 0xa580BBCB1Cee2BCec4De2Ea870D20a12A964819e;
+
+    address immutable internal STARKNET_GOV_RELAY = DssExecLib.getChangelogAddress("STARKNET_GOV_RELAY");
+    address constant internal NEW_STARKNET_GOV_RELAY = 0x2385C60D2756Ed8CA001817fC37FDa216d7466c0;
+    uint256 constant internal L2_GOV_RELAY_SPELL = 0x013c117c7bdb9dbbb45813fd6de8e301bbceed2cfad7c4c589cafa4478104672;
```

Comment:
@maciejka please sign-off by approving the changes once all the sanity checks and reviews are carried out by SNE Core Unit, especially the L2 spel, as per PR checklist:
`Notify any external teams affected by the spell so they have the opportunity to review`

Reply:
Starknet related changes LGTM.